core/src/test/scala/org/apache/spark/util/UtilsSuite.scala

```diff
@@ -473,7 +474,7 @@ class UtilsSuite extends SparkFunSuite with ResetSystemProperties with Logging {
     val rawCwd = System.getProperty("user.dir")
     val cwd = if (Utils.isWindows) s"/$rawCwd".replace("\\", "/") else rawCwd
     assertResolves("hdfs:/root/spark.jar", "hdfs:/root/spark.jar")
-    assertResolves("hdfs:///root/spark.jar#app.jar", "hdfs:/root/spark.jar#app.jar")
+    assertResolves("hdfs:///root/spark.jar#app.jar", "hdfs:///root/spark.jar#app.jar")
```

Comment:
I think we do use it in `assert(new URI(Utils.resolveURIs(before)) === new URI(after))`. 

> different cases of `Utils.resolveURI`

I think we should identify the cases.

If the only reason is just meant to make the test code prettier but change the existing tests for no reason, I would go -1.